app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, file, file_to_copy = command.split()
+    if command_name != "cp":
+        print("This command not exist")
+        return
+    try:
+        with (open(file, "r") as current_file,
```

Comment:
You seem to have forgotten to check for this condition:
`It must do nothing if the user tries to copy the file to file with the same.`

Reply:
I don't, I just use mode x  — writes data to a file and returns an error if the file already exists (exclusive creation); So it's can't copy file with the same name